ISHLogDNAService/ISHLogDNAService.m

```diff
@@ -79,6 +82,20 @@ + (instancetype)messageWithLine:(NSString *)line level:(ISHLogDNALevel)level met
         [metaWithVersion setObject:buildNumber forKey:ISHLogDNAServiceKeyBundleVersion];
     }
 
+    struct utsname systemInfo;
+    uname(&systemInfo);
+    NSString *deviceName = [NSString stringWithCString:systemInfo.machine encoding:NSUTF8StringEncoding];
+
+    if (deviceName.length) {
+        [metaWithVersion setObject:deviceName forKey:ISHLogDNAServiceKeyModelName];
+    }
+
+    NSString *systemVersion = NSProcessInfo.processInfo.operatingSystemVersionString;
+
+    if (systemVersion.length) {
+        [metaWithVersion setObject:systemVersion forKey:ISHLogDNAServiceKeySystemVersion];
+    }
```

Comment:
Would it make sense to wrap the above in a `dispatch_once`. And then create a mutable copy per message? I would assume that the system/app related properties won't change anyway per app lifecycle.